static/js/stream_create.js

```diff
@@ -430,25 +431,26 @@ export function set_up_handlers() {
         }
 
         if (principals.length >= 50) {
-            const invites_warning_modal = render_subscription_invites_warning_modal({
+            const modal_parent = $("#subscription_overlay");
+            const html_body = render_subscription_invites_warning_modal({
                 stream_name,
                 count: principals.length,
             });
-            $("#stream-creation").append(invites_warning_modal);
```

Comment:
Any reason for changing the parent element of this modal?

Reply:
Added a comment <a href = 'https://github.com/zulip/zulip/pull/18629#issuecomment-857016235'>here</a>.